packages/plugin-core/src/KeybindingUtils.ts

```diff
@@ -139,4 +139,27 @@ export class KeybindingUtils {
     migratedKeybindings = assign(keybindings, migratedKeybindings);
     return { keybindingConfigPath, migratedKeybindings };
   }
+
+  /**
+   * For the given pod ID, returns a user-configured shortcut (in VSCode
+   * settings) if it exists. Otherwise, returns undefined.
+   * @param podId
+   * @returns
+   */
+  static getKeybindingForPodIfExists(podId: string) {
+    const { keybindingConfigPath } = this.getKeybindingConfigPath();
+    const keybindings: Array<any> =
+      readJSONWithCommentsSync(keybindingConfigPath);
+
+    const result = keybindings.filter(
+      (item) =>
+        item.command &&
+        item.command === DENDRON_COMMANDS.EXPORT_POD_V2.key &&
+        item.args === podId
+    );
+
+    if (result.length === 1 && result[0].key) {
```

Comment:
can we show an error message if there are multiple results?

Reply:
Will adjust to having the quick pick detail display 'Multiple Keybindings' instead.